spec/system/task_spec.rb

```diff
@@ -1,27 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
Factoryにassociationを定義するとproject_idの指定は省略できますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
2468844 で修正しました。